extensions/job_management/standard/requirements/job-management/update/REQ_patch-op.adoc

```diff
@@ -0,0 +1,8 @@
+[[req_job-management_update_patch-op]]
+[requirement]
+====
+[%metadata]
+label:: /req/job-management/update-patch-op
+part:: For every created jobs (path '/jobs/{jobId}'), the server SHALL support the HTTP PATCH operation.
+part:: The parameter 'jobId' is each 'jobID' property in the jobs list response (JSONPath: `$.jobs[*].id`).
+====
```

Comment:
In openEO the PATCH operation is optional. Many implementations struggle a bit with implementing updating jobs, especially when already executed. I think I'd recommend to not require this endpoint and make it optional. Could be explored again via a link in GET /jobs/:id (we don't have that link defined in openEO yet, but happy to add).

Reply:
I agree about making it optional. This is an advanced feature that is not mandatory.

However, one thing this proposal does is that it limits PATCH only to jobs that have not started (and by "started", I mean they must be in `status: created` explicitly; async job queued in `status: accepted` or any [core job status](https://github.com/opengeospatial/ogcapi-processes/blob/3914c8a3d000c06df805e799bd50af9e43fc2d29/openapi/schemas/processes-core/statusCode.yaml) is NOT updatable).

Reply:
just informational: openEO also allows to update finished and errored jobs (e.g. update the title of the job). Active jobs (queued and running) we also reject changes for.